app/products/calls/utils.ts

```diff
@@ -201,3 +203,28 @@ export function isCallsStartedMessage(payload?: NotificationData) {
     // calls will be >= 0.21.0, and push proxy will be >= 5.27.0
     return (payload?.message === 'You\'ve been invited to a call' || callsMessageRegex.test(payload?.message || ''));
 }
+
+export const hasCaptions = (postProps?: Record<string, any> & { captions?: Caption[] }): boolean => {
+    return !(!postProps || !postProps.captions?.[0]);
+};
+
+export const getTranscriptionUri = (serverUrl: string, postProps?: Record<string, any> & { captions?: Caption[] }): {
+    track?: SubtitleTrack;
+    selected?: SelectedSubtitleTrack;
+} => {
+    // Note: We're not using hasCaptions above because this tells typescript that the caption exists later.
+    // We could use some fancy typescript to do the same, but it's not worth the complexity.
+    if (!postProps || !postProps.captions?.[0]) {
```

Comment:
I think we can more simply check for `postProps.captions?.length > 0`?

Reply:
Not in typescript, unfortunately -- the expression might be undefined, so can't compare it to 0.